internal/report/report.go

```diff
@@ -1070,6 +1071,16 @@ func reportLabels(rpt *Report, g *graph.Graph, origCount, droppedNodes, droppedE
 		flatSum = flatSum + n.FlatValue()
 	}
 
+	if len(rpt.options.ActiveFilters) > 0 {
+		label = append(label, "Active filters:")
+		for i, s := range rpt.options.ActiveFilters {
+			if len(s) > 80 {
+				rpt.options.ActiveFilters[i] = s[:70] + "...(regex continues)"
```

Comment:
Maybe use the unicode ellipsis character (…) and drop the "(regex continues)" part. Using the ellipsis should make things clear enough that these dots are not part of the regex but rather represent the omission.